content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,301 @@
+
+![Light grey text passing AA for large text, but not small text](./pass_aa_large.png)
+
+In this example you can see that the text passes the WCAG AA requirements for large text, but fails the same requirements for small text.
+
+# Fantastic Fonts {#font-resize}
+
+One of the most widely used accessibility features is font scaling. While many browsers default to a font size of `16px`, the user is actually able to change settings on their device to configure websites to use a larger font size.
+
+Many phones using iOS and Android allow users to change the font size on their mobile devices. This feature is heyso commonplace that it often prompts users to change this setting when the phone is being set up.
+
+<div style="display: flex; justify-content: space-around">
+    <figure>
+		<img src="./ios_text_size.png" style="height: 300px" alt=""/>
+        <figcaption>
+            iOS font size settings screen
+        </figcaption>
+    </figure>
+    <figure>
+        <img src="./android_text_size.png" style="height: 300px" alt=""/>
+        <figcaption>
+            Android font size settings screen
+        </figcaption>
+    </figure>
+</div>
+
+Not only do you have these settings on mobile devices, but they're available on desktop as well. 
+
+Using Chrome, go to [your settings page](chrome://settings/?search=font+size), and you should be able to set your font size.
+
+![Font settings in Chrome](./chrome_font_size.png)
+
+You can do the same in Firefox in [your preferences](about:preferences#general).
+
+![Font settings in Firefox](./firefox_font_size.png)
+
+
+## Implementation {#font-rem}
+
+While browsers have the ability to set the font size, if you're using `px`, `vw`, `vh`, or other unit values for your fonts, the browser will not update these font sizes for you. In order to have your application rescale the font size to match the browser settings, you'll need to use the `rem` unit.
+
+You can think of `rem` as a multiplier to apply to the default font size. When the browser's font size is set to `16px`:
+
+- `1rem` will be `16px` (1 * 16px)
+- `1.5rem` will be `24px` (1.5 * 16px)
+- `3rem` will be `48px` (3 * 16px)
+
+Likewise, when the browser's font size is set to `20px`:
+
+- `1rem` will be `20px` (1 * 20px)
+- `1.5rem` will be `30px` (1.5 * 20px)
+- `3rem` will be `60px` (3 * 20px)
+
+> Something to keep in mind is that `rem` is a _relative_ font size. It's relative to the root element's font size. _This means that you cannot set a default `px` value font size in `:root`  , as it will disable font scaling, even if the rest of your page is using `rem` values._
+
+It's highly encouraged to keep font sizes standardized around the `1rem` value. While this may seem like a frustrating limitation at first, see it from the user's perspective.
```

Comment:
Font sizes can be larger, e.g. for heading or callouts. Maybe say body text size or minimum?

Reply:
I originally had the language "minimum", but changed it for various reasons. I'll change to "body text size" as I think that fits better and call out that they can be larger

Reply:
```suggestion
It's highly encouraged to keep body text size standardized around the `1rem` value. While this may seem like a frustrating limitation at first, see it from the user's perspective. Font sizes can, of course, be larger than this for things like headers or callouts, but the primary content of your site should default to this sizing.
```